types/pathval/tslint.json

```diff
@@ -0,0 +1,15 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
New packages should not override any lint rules. They should only extend `dtslint/dt.json`.